src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/BookingTest.kt

```diff
@@ -1215,12 +1215,26 @@ class BookingTest : IntegrationTestBase() {
           )
           .exchange()
           .expectStatus()
-          .is4xxClientError
+          .isOk
```

Comment:
Test title should be updated to `... returns OK...` instead of `... returns 409 Conflict...` ([line 1155](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1737/files#diff-814a441d05bc78abafaaa017a1339bc6150666556a526da2ae6665e187a2b9f7R1155))

Reply:
Its my fault, fixed it.